README.md

```diff
@@ -1,75 +1,72 @@
-# QHub
+# `qhub` - automated data science environments on cloud environments
 
-Is a tool for initializing and maintaining the state of QHub
-deployments on Digital Ocean, Amazon Web Services, and Google Cloud
-Platform.
+:badges:
 
-## Installation:
+[`qhub`][qhub] is an opinionated deployment of open source data science infrastrcuture (eg. [jupyterhub], [dask], and [kubernetes]) on different cloud service providers without vendor lock-in. [`qhub`][qhub] is easy to install and maintain.
 
-```bash
-pip install qhub
-```
+## Usage
 
-## Configuration
+`qhub` is install as a command line application in python. it requires you to choose your the cloud provider you desire.
```

Comment:
install -> installed
python -> Python
it -> It

Something along these lines
it requires you to choose your the cloud provider you desire. -> It can be installed on a cloud provider of your choice.